packages/manifest-helper/src/manifest.ts

```diff
@@ -52,6 +52,7 @@ const validateSchema = (manifest: Manifest): void => {
     throw new Error(`The 'options' property should be an array`)
   }
 
+  //TODO: update this to use the new validation functions
```

Comment:
Hey @demetriusfeijoo I also found this, do you remember why we had the same code twice here ?

Reply:
Sorry Bibi, I didn't notice this comment from you. 😞 

I guess now you already figured it out but they were duplicated because the `manifest-helper` was created while working on the manifest on the CLI but the `helper/vite` wasn't refactored to use it yet so it was duplicated there.

By the way, the work you did here will close also the SHAPE-5376 ticket. 🚀 🚀

Reply:
thank you so much for the hint <3